x/migration/keeper/morse_account_state.go

```diff
@@ -0,0 +1,39 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/store/prefix"
+	"github.com/cosmos/cosmos-sdk/runtime"
+
+	"github.com/pokt-network/poktroll/x/migration/types"
+)
+
+// SetMorseAccountState sets morseAccountState in the store
+func (k Keeper) SetMorseAccountState(ctx context.Context, morseAccountState types.MorseAccountState) {
+	storeAdapter := runtime.KVStoreAdapter(k.storeService.OpenKVStore(ctx))
+	store := prefix.NewStore(storeAdapter, types.KeyPrefix(types.MorseAccountStateKey))
+	morseAccountStateBz := k.cdc.MustMarshal(&morseAccountState)
+	store.Set([]byte{0}, morseAccountStateBz)
```

Comment:
Could we extract the `[]byte{0}` key into const?

Reply:
I don't see a clean way to do this since slices (end even arrays) can't be constants. :thinking: